deploy/api.yaml

```diff
@@ -0,0 +1,157 @@
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Error"
+      x-amazon-apigateway-request-validators: "Validate body"
+      x-amazon-apigateway-integration:
+        type: "aws_proxy"
+        httpMethod: "POST"
+        uri:
+          Fn::Sub: "arn:aws:apigateway:${AWS::Region}:lambda:path/2015-03-31/functions/${KBVSessionFunction.Arn}/invocations"
+        passthroughBehavior: "when_no_match"
+  /question:
+    get:
+      parameters:
+        - $ref: "#/components/parameters/SessionHeader"
+      responses:
+        "200":
+          description: "200 response"
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Question"
+        "201":
+          description: "201 response"
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Question"
+        "400":
+          description: "400 response"
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Error"
+        "500":
+          description: "500 response"
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Error"
+      x-amazon-apigateway-request-validators: "Validate body"
+      x-amazon-apigateway-integration:
+        type: "aws_proxy"
+        httpMethod: "POST"
+        uri:
+          Fn::Sub: "arn:aws:apigateway:${AWS::Region}:lambda:path/2015-03-31/functions/${KBVQuestionFunction.Arn}/invocations"
+        passthroughBehavior: "when_no_match"
+  /answer:
+    post:
+      parameters:
+        - $ref: "#/components/parameters/SessionHeader"
+      responses:
+        "201":
+          description: "201 response"
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Answer"
+        "400":
+          description: "400 response"
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Error"
+        "500":
+          description: "500 response"
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Error"
+      x-amazon-apigateway-request-validators: "Validate body"
+      x-amazon-apigateway-integration:
+        type: "aws_proxy"
+        httpMethod: "POST"
+        uri:
+          Fn::Sub: "arn:aws:apigateway:${AWS::Region}:lambda:path/2015-03-31/functions/${KBVAnswerFunction.Arn}/invocations"
+        passthroughBehavior: "when_no_match"
+components:
+  parameters:
+    SessionHeader:
+      name: session-id
+      in: header
+      description: A UUID generated by the KBV API to act as a primary key for the KBVSessionTable in DynamoDB
+      required: true
+      schema:
+        type: string
+  schemas:
+    Session:
+      type: "object"
+      properties:
+        session-id:
```

Comment:
We use `kebab-case` for `session-id` and `lowerCamelCase` for `questionId`.

Which one should we use for consistency?

Reply:
I think they are both different objects of Http request. `session-id` is part of the `request header` and `questionId` is part of the POST `request body` payload. 
When we are using the request body object (questionId) and it is a Java object which will be marshalled during conversion to a JSON object. Java variables cannot be declared with a hyphen (-). We use lowerCamelCase for variable declaration in many places.